policy-test/tests/api.rs

```diff
@@ -331,3 +319,20 @@ fn mk_admin_server(ns: &str, name: &str) -> k8s::policy::Server {
         },
     }
 }
+
+async fn retry_watch_server(
+    client: &kube::Client,
+    ns: &str,
+    pod_name: &str,
+) -> tonic::Streaming<grpc::inbound::Server> {
+    // Port-forward to the control plane and start watching the pod's admin
+    // server's policy and ensure that the first update uses the default
+    // policy.
+    let mut policy_api = grpc::PolicyClient::port_forwarded(client).await;
+    loop {
```

Comment:
i don't necessarily think that we need to add a really fully-featured retry implementation with jitter and exponential backoffs, or anything like that...but, do you think we should maybe enforce _some_ kind of limit on how many times we'll retry here, so that if something else is actually broken, the test won't loop infinitely?

Reply:
could add a timeout, but my thinking was that github actions would handle timing things out for us if this was in a permanent failure state.

Reply:
yeah, fair enough. now that we're logging the error, i'm less concerned about impossible-to-diagnose test hangs.